azure-communication-ui/calling/src/main/res/layout/azure_communication_ui_calling_fragment_setup.xml

```diff
@@ -240,33 +240,31 @@
             app:layout_constraintTop_toTopOf="parent"
             />
 
-        <androidx.constraintlayout.widget.ConstraintLayout
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            app:layout_constraintBottom_toBottomOf="parent"
+        <LinearLayout
```

Comment:
I'm a bit sus of these layout_constraints w/ a linear layout.

internal class JoinCallButtonHolderView : ConstraintLayout

You should be able to flatten this. There should be no need for a linear layout or constraint layout embedded in this fashion into a ConstraintLayout.

Reply:
Removed LinearLayout